src/obis-observer/commands/downlink/SetupMeter.ts

```diff
@@ -31,26 +34,28 @@ const examples: TCommandExampleList = [
  *
  * @example
  * ```js
- * import GetShortNameProfile from 'jooby-codec/obis-observer/commands/downlink/GetShortNameProfile.js';
+ * import SetupMeter from 'jooby-codec/obis-observer/commands/downlink/SetupMeter.js';
  *
  * const parameters = {
- *     requestId: 4,
- *     shortName: 128
+ *     requestId: 3,
+ *     meterId: 20,
+ *     meterProfileId: 17,
+ *     address: 'ma2375'
  * };
- * const command = new GetShortNameProfile(parameters);
+ * const command = new SetupMeter(parameters);
  *
  * // output command binary in hex representation
  * console.log(command.toHex());
- * // 09 04 80
+ * // 70 0a 03 14 11 06 6d 61 32 33 37 35
  * ```
  *
- * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/obis-observer/commands/GetShortNameProfile.md#request)
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/obis-observer/commands/SetupMeter.md#request)
  */
-class GetShortNameProfile extends Command {
-    constructor ( public parameters: IGetShortNameProfileParameters ) {
+class SetupMeter extends Command {
+    constructor ( public parameters: ISetupMeterParameters ) {
         super();
 
-        this.size = COMMAND_SIZE;
+        this.size = REQUEST_ID_SIZE + 3 + parameters.address.length;
```

Comment:
maybe put it to COMMAND_SIZE const?

Reply:
the size of this command is not const and static